README.md

```diff
@@ -1,6 +1,7 @@
 # Bundle Update Buildkite Plugin
 
-![Build status](https://badge.buildkite.com/3e169bf223b02ed62d3a3090e2ef78cb025e83076c33b694b5.svg?branch=main)
+[![tests](https://github.com/envato/bundle-update-buildkite-plugin/actions/workflows/tests.yml/badge.svg?branch=main)](https://github.com/envato/bundle-update-buildkite-plugin/actions/workflows/tests.yml)
+
```

Comment:
Let's remove this empty line so the badges end up on the same line when rendered. It currently looks like this:

<img width="486" alt="image" src="https://user-images.githubusercontent.com/497874/152704595-616c0217-54be-44ea-ac2c-bfd46cb7b428.png">

```suggestion
```